README.md

```diff
@@ -1 +1,5 @@
 # minimit-llm
```

Comment:
You should have resolved this conflict. I guess, its not a bit deal, just remove conflict artefacts

Reply:
<img width="923" height="310" alt="image" src="https://github.com/user-attachments/assets/15c876ec-d060-48bb-8b27-f58535e0b61e" />